Сетка страниц и крупных блоков на флексах

Nemchenco96 opened the pull request:
Выполнил стилизацию и добавил флексы. Единственное, в некоторых местах не могу центрировать элементы. Как я понимаю, это дальше будет?

---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/35/user/2065035/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/2065035-sedona-35/4/

Nemchenco96 committed:
4 и 5 тема

Выполнил стилизацию и добавил флексы. Единственное, в некоторых местах не могу центрировать элементы. Как я понимаю, это дальше будет?

Dima-Dim commented:
> Выполнил стилизацию и добавил флексы. Единственное, в некоторых местах не могу центрировать элементы. Как я понимаю, это дальше будет?

Да, дальше ещё должна быть информация по стилизации. Если в демках элементы не центровали, то будет дальше.

Dima-Dim commented on catalog.html:
```diff
@@ -59,35 +59,36 @@ <h1 class="inner-header-title">Гостиницы Седоны</h1>
+
         <section class="list-of-hotels">
-         <h2 class="visually-hidden">Критерии выбора гостиницы</h2>
+         <h2 class="visually-hidden" hidden>Критерии выбора гостиницы</h2>
```
Зачем тут два скрывающих класса.

Dima-Dim commented on styles/styles.css:
```diff
@@ -30,263 +30,403 @@
+
+.main-text {
+  display: block;
+  height: 282px;
```
Возможно тоже ещё не было информации, но так ограничивать высоту у текстовых блоков не нужно, это приведёт к проблемам при переполнении. Не забудь потом это исправить.

Dima-Dim commented on styles/styles.css:
```diff
@@ -30,263 +30,403 @@
-font-family: "PT Sans", sans-serif;
-color: #000000;
-background-color: rgba(131, 179, 211, 0.12);
+  display: block;
```
Тут точно нужно указывать это свойство? Если не указать, то что не так?

Dima-Dim commented on styles/styles.css:
```diff
@@ -30,263 +30,403 @@
+  background-color: rgba(131, 179, 211, 0.12)
 }
 
 .advantages-item-road {
```
Для чего нам столько одинаковых классов?

Nemchenco96 committed:
Устранение замечаний

1) Оставил один hidden в разметке для себя, что бы не вываливалось из блока с критерием выбора в каталоге (стр 63)
2) Лишние свойства убрал
3) Одинаковые классы почистил

Dima-Dim commented on catalog.html:
```diff
@@ -123,15 +123,14 @@ <h2 class="visually-hidden">Критерии выбора гостиницы</h2
-      <section class="searching results">
 
+      <section class="searching results">
+        <h3 class="amount">Найдено гостиниц:</h3>
```
Хотя бы `amount-title` в теге ведь нет количества.

Dima-Dim commented on index.html:
```diff
@@ -45,50 +45,53 @@
+      </div>
+
+      <section class="main-text">
+        <h1 class="main-text-sedona">СЕДОНА-НЕБОЛЬШОЙ ГОРОДОК В АРИЗОНЕ, ЗАСЛУЖИВАЮЩИЙ БОЛЬШЕГО!</h1>
```
Для стилизации текста есть css.

Dima-Dim commented on styles/styles.css:
```diff
@@ -30,263 +30,366 @@
+.main-text-advantages{
+  display: block;
+  width: 651px;
+  height: 26px;
```
Не забудь убрать фиксированную высоту у всех тегов в которых есть текст на любом из уровней вложенности.

Dima-Dim commented on styles/styles.css:
```diff
@@ -30,263 +30,366 @@
+  margin-top: 35px;
+  font-size: 60px;
+  line-height: 78px;
+  font-weight: 700;
```
`font-weight` тоже можно указать для какого-то из корневых тегов и где во вложенных уже не указывать его.

Dima-Dim commented on styles/styles.css:
```diff
@@ -30,263 +30,366 @@
+  background-color: #ffffff;
 }
 
+.searching results {
```
Что за такой селектор `.searching results`?

Dima-Dim reviewed (changes requested):
validator.w3.org/nu/

Nemchenco96 committed:
Исправление -3

1) Название тега amount поменял на amount-title;
2) Убрал весь текст, который написан капслоком, указал все необходимое в стилях
3) Фиксированную высоту у текста убрал везде
4) font-weight указал для корневых, где нашел. Лишнее удалил, вроде бы во вложенных. Надеюсь не пропустил нигде
4) Ошибку в написании класса .searching-results исправил.
Также, добавил пару div-ов на будущие темы. Надеюсь все окей:0

Dima-Dim reviewed (commented):
> добавил пару div-ов на будущие темы

Нет :) не нужно ничего добавлять "на будущее". Потом, не факт что оно пригодится и если не пригодится, то высока вероятность забыть это убрать.
